internal/config/config_test.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestNewConfig(t *testing.T) {
+	tests := []struct {
+		name    string
+		input   map[string]string
+		want    Config
+		wantErr bool
```

Comment:
You should probably define returned errors in `config.go` var section
```
var (
    errMissingPort = errors.New("port is not defined in environment")
)
```
and use them here in test, by setting expected errors and checking them while testing. Something like

````
if (tt.expectedErr != nil)  && !errors.Is(err, errMissingPort) {
    ...
}
```

Reply:
thank you, didn't think about this way, couldn't find way how to make work `errors.Is`

Reply:
fixed